zproject/prod_settings_template.py

```diff
@@ -669,6 +669,10 @@
 ## to '' will disable the Camo integration.
 CAMO_URI = "/external_content/"
 
+## The new user tutorial is enabled by default, but can be disabled for
+## self-hosters who want to disable the tutorial entirely on their system.
+# TUTORIAL_ENABLED = True
```

Comment:
Hmm, this might be an option that we expect to only be used by folks who are more or less forking Zulip, in which case I'd want to not document it here.

@alexmv @andersk what are your thoughts?

Reply:
Eh, I think it's fine to document. I don't feel like it's all that much of a footgun.